nixos/tests/all-tests.nix

```diff
@@ -54,10 +54,9 @@ in
 builtins.mapAttrs
   (
     name: value:
-    nixos-lib.runTest (
+    pkgs.testers.runNixOSTest (
```

Comment:
Oh nice, didn't notice (or didn't remember about) `testers.runNixOSTest`. Looks like it's partially applied, so we can drop `hostPkgs` below